CHANGELOG.md

```diff
@@ -71,6 +71,7 @@ This release containing major architectural changes, new platform support, enhan
 
 
 ### Added
+- Add manual type inference feature due to ambiguous inference when test inputs are not representing the type well enough
```

Comment:
Please rebase your PR and add the link to the PR to the list of PRs.

Reply:
This is addressed in https://github.com/pulp-platform/Deeploy/pull/68/commits/bdd8ae5390c2426a88eeaf364e55f90259014b1a